src/lib/__tests__/util-test.js

```diff
@@ -397,20 +397,17 @@ describe('lib/util', () => {
                 assert.ok(head.querySelector('script[src="bar"]') instanceof HTMLScriptElement);
             });
 
-            it('should clear requireJS until scripts are loaded or fail to load', () => {
-                window.define = true;
-                window.require = true;
-                window.requrejs = true;
-
-                return util.loadScripts(['foo', 'bar'], true).catch(() => {
-                    expect(window.define).to.equal(true);
-                    expect(window.require).to.equal(true);
-                    expect(window.requirejs).to.equal(true);
-                });
+            it('should disable AMD until scripts are loaded or fail to load', () => {
+                const func = () => {};
+                func.amd = ['jquery'];
+                window.define = func;
+
+                const promise = util.loadScripts(['foo', 'bar'], true);
+                expect(define).to.equal(undefined);
 
-                expect(window.define).to.equal(undefined);
-                expect(window.require).to.equal(undefined);
-                expect(window.requirejs).to.equal(undefined);
```

Comment:
FYI @jeremypress, this test didn't check anything before. These last 3 statements are after the return and weren't checked. The expect statements within the catch block were also not checked there was there was no exception to catch.